### ISSUE TITLE
Example for testing

### ISSUE THREAD

**Issue by maehler** [PlantGenIE/exheatmap#1]
In its current state there seem to be a lot of references to files that do not exist, and the application is not loading as it should. I'm running it with

```bash
php -S localhost:8080
```

and this is the log output when loading it in the browser:

```
exHeatmap.php:3 GET http://localhost:8080/plugins/tour/css/poptour.css net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:4 GET http://localhost:8080/plugins/exheatmap/css/explot.css net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:16 GET http://localhost:8080/plugins/exheatmap/javascript/kinetic-v5.1.0.min.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:17 GET http://localhost:8080/plugins/exheatmap/javascript/inchlib-1.2.0.min.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:18 GET http://localhost:8080/plugins/exheatmap/javascript/jspdf.min.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:48 GET http://localhost:8080/plugins/exheatmap/javascript/init.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:75 GET http://localhost:8080/plugins/exdraw/js/jquery.tabSlideOut.v1.3.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:222 GET http://localhost:8080/plugins/tour/poptour.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:223 GET http://localhost:8080/plugins/tour/exheatmap.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:224 GET http://localhost:8080/plugins/tour/workflow.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:20 GET http://localhost:8080/plugins/gene/css/load1.gif 404 (Not Found)
exHeatmap.php:4 GET http://localhost:8080/plugins/exheatmap/css/explot.css net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:16 GET http://localhost:8080/plugins/exheatmap/javascript/kinetic-v5.1.0.min.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:17 GET http://localhost:8080/plugins/exheatmap/javascript/inchlib-1.2.0.min.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:18 GET http://localhost:8080/plugins/exheatmap/javascript/jspdf.min.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:48 GET http://localhost:8080/plugins/exheatmap/javascript/init.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:75 GET http://localhost:8080/plugins/exdraw/js/jquery.tabSlideOut.v1.3.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:77 Uncaught ReferenceError: $ is not defined
    at exHeatmap.php:77
exHeatmap.php:222 GET http://localhost:8080/plugins/tour/poptour.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:1 GET http://localhost:8080/plugins/exdraw/img/sidebar_background.png 404 (Not Found)
exHeatmap.php:1 GET http://localhost:8080/plugins/exdraw/img//right_sidebar3.png 404 (Not Found)
exHeatmap.php:223 GET http://localhost:8080/plugins/tour/exheatmap.js net::ERR_ABORTED 404 (Not Found)
exHeatmap.php:224 GET http://localhost:8080/plugins/tour/workflow.js net::ERR_ABORTED 404 (Not Found)
favicon.ico:1 GET http://localhost:8080/favicon.ico 404 (Not Found)
```

It seems that it depends on some different plugins, but it also seems that paths to exheatmap resources are hardcoded. Would it be possible to ditch dependencies to other plugins and to have a test dataset (either integrated into the repository, or an API call) that can be used in the development process?

**Comment by irusri** [comment]
I added the JS and CSS files from other tools into the exHeatmap directory. However, this is still not working due to the Python libraries that we used for [Inchlib library]( https://www.openscreen.cz/software/inchlib/home/).

I uploaded the latest [version of exHeatmap](https://github.com/PlantGenIE/exheatmap-1) tool that we used in beta sites into Github. This should work with the PHP server as it uses the API.
